developer_manual/core/theming.rst

```diff
@@ -1,166 +1,183 @@
+Shared with you/others image                                     ``owncloud/core/img/share.svg``
+Shared by link image                                             ``owncloud/core/img/public.svg``
+Tags image                                                       ``owncloud/core/img/tag.svg``
+Deleted files image                                              ``owncloud/core/img/delete.svg``
+Settings image                                                   ``owncloud/core/img/actions/settings.svg``
+Search image                                                     ``owncloud/core/img/actions/search-white.svg``
+Breadcrumbs home image                                           ``owncloud/core/img/places/home.svg``
+Breadcrumbs separator                                            ``owncloud/core/img/breadcrumb.svg``
+Dropdown arrow                                       Admin Menu  ``owncloud/core/img/actions/caret.svg``
+Personal image                                                   ``owncloud/settings/img/personal.svg``
+Users image                                                      ``owncloud/settings/img/users.svg``
+Help image                                                       ``owncloud/settings/img/help.svg``
+Admin image                                                      ``owncloud/settings/img/admin.svg``
+Logout image                                                     ``owncloud/core/img/actions/logout.svg``
+Apps menu - Files image                                          ``owncloud/apps/files/img/app.svg``
+Apps menu - Plus image                                           ``owncloud/settings/img/apps.svg``
+The favicon                                                      ``owncloud/apps/theme-example/core/img/favicon.ico``  
+Upload image                                         Personal    ``owncloud/core/img/actions/upload.svg``
+Folder image                                                     ``owncloud/core/img/filetypes/folder.svg``
+Trash can image                                                  ``owncloud/core/img/actions/delete.svg``
+==================================================== =========== ====================================================
+
+.. note:: 
+   When overriding the favicon, make sure your custom theme includes and override for both ``owncloud/apps/core/img/favicon.svg`` and ``owncloud/apps/core/img/favicon.png``, to cover any future updates to favicon handling.
+
+How to Override the Default Colors
+----------------------------------
+
+To override the default style sheet, create a new CSS style sheet in your theme, in the theme’s ``css`` directory, called ``styles.css``.
+
+To override the background in the login page and the blue header bar on the main navigation page, visible once you log in to ownCloud, you need to override the CSS ``body-login`` element , which you can see the default definition of below. 
```

Comment:
`body-login` is an id. So you could say "you need to define styles for the html-tag with the `body-login` id."